crates/rome_js_syntax/src/static_value.rs

```diff
@@ -98,8 +10,10 @@ pub enum StaticValue {
     Undefined(JsSyntaxToken),
     Number(JsSyntaxToken),
     BigInt(JsSyntaxToken),
-    String(QuotedString),
-    TemplateChunk(Option<JsSyntaxToken>),
+    /// The string can be empty.
+    String(JsSyntaxToken),
+    /// This is use to represent empty template strings.
```

Comment:
```suggestion
    /// This is used to represent empty template strings.
```

Is this valid also for cases like the following?

```js
`${variable}`
```

Technically it's "empty", in the sense that has empty strings inside.

Reply:
No, this is just valid for the empty template string (without any element). I should rephrase to `the empty template string`